### PR TITLE
KOJO-172 | UseCase50 for 5.0 release

### PR DESCRIPTION
**Pull request by mucha55**
* Did some cleanup in this UseCase because there was a lot of unnecessary cruft
* Added `data/*/dumps/kojo_fitness.sql` since it seemed like an unnecessary step to require users to do it
* UseCase50 sometimes succeeds, sometimes fails, and the logs show a) the updated log message structure, and b) panicking triggered by both uncaught errors and exceptions

### CODE REVIEW

File: UseCase50/composer.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "email": "brad.wilson@neighborhoods.com"
+    }
+  ],
+  "config": {
+    "sort-packages": true
+  },
+  "require": {
+    "php": ">=7.2",
+    "neighborhoods/kojo": "5.x-dev",
+    "symfony/console": "4.3.*"
```
Comment (mucha55) [inline UseCase50/composer.json]: Symfony Console 4.4 was causing a problem, this is just a workaround (kojo proper will fix it itself)

File: UseCase50/src/V1/Worker.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        } else {
+            $this->getApiV1WorkerService()->getLogger()->info(
+                'Userspace failing with uncaught error',
+                [
+                    'job_type_code' => Worker\Facade::JOB_TYPE_CODE,
+                    'event_type' => 'userspace_failing_uncaught_error',
+                    'job_id' => $this->getApiV1WorkerService()->getJobId()
+                ]
+            );
+            throw new \Error('Uncaught userspace exception');
```
Comment (alexberryman) [inline UseCase50/src/V1/Worker.php]:
```suggestion
            throw new \Error('Uncaught userspace error');
```

File: UseCase50/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Setup UseCase50
+
+### Running Kōjō
+Setting up the containers using docker-compose:
+
+```bash
+# Get the latest repo
+git clone git@github.com:neighborhoods/KojoFitness.git;
+cd KojoFitness;
+git checkout 4.x;
```
Comment (alexberryman) [inline UseCase50/README.md]:
```suggestion
git checkout 5.x;
```

Comment (mucha55) [reply UseCase50/README.md]: It should actually be `master` like we discussed offline, I've pushed the change

**Pull request merged**